lib/user.rb

```diff
@@ -0,0 +1,37 @@
+class User
+  attr_reader :id, :name, :username, :email, :password
+
+  def initialize(id:, name:, username:, email:, password:)
+    @id = id
+    @name = name
+    @username = username
+    @email = email
+    @password = password
+  end
+
+  def self.sign_up(name:, username:, email:, password:)
+    user = connect_to_database.exec "INSERT INTO users (name, username, email, password) VALUES('#{name}', '#{username}', '#{email}', '#{password}') RETURNING id, name, username, email, password;"
```

Comment:
I think that will be best to keep the SQL query inside () so you are consistent with the rest of your code.